src/sidebar/components/svg-icon.js

```diff
@@ -7,6 +7,7 @@ const propTypes = require('prop-types');
 // The list of supported icons
 const icons = {
   add: require('../../images/icons/add.svg'),
+  cancel: require('../../images/icons/cancel.svg'),
```

Comment:
I know there has already been discussion of an immediate iteration on this design that gets rid of the `X` icon, but we'll need that icon for other things soon anyway.